packages/extension-polkagate/src/fullscreen/manageProxies/Review.tsx

```
@@ -5,15 +5,16 @@ import type { SubmittableExtrinsic } from '@polkadot/api/types';
 import type { Balance } from '@polkadot/types/interfaces';
 import type { ISubmittableResult } from '@polkadot/types/types';
 import type { BN } from '@polkadot/util';
-import type { Proxy, ProxyItem, TxInfo } from '../../util/types';
+import type { CanPayFee, Proxy, ProxyItem, TxInfo } from '../../util/types';
```

Biome (2.1.2)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)